dist/lib/util/labelUtils.d.ts

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
Do we need the extra file or can we convert the utilities to a ts file?

Reply:
will check this